.changeset/strange-olives-count.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
This is how changesets works. We no longer go by commits but by changesets. We can stack changesets up in a PR or just not have any. There is a github bot that will post a comment saying "hey i found a changeset" or "no changesets found"

This is how we can identify a PR that will release. 

We also can edit these after the fact or before. Its not a big deal to not edit them. The naming is automatically done by changesets to have unique id's.

Reply:
Correct, but we can edit them at any time